packages/flutter_tools/lib/src/commands/build.dart

```diff
@@ -8,12 +8,14 @@ import 'dart:io';
 import '../globals.dart';
 import '../runner/flutter_command.dart';
 import 'build_apk.dart';
+import 'build_ios.dart';
 import 'build_flx.dart';
 
 class BuildCommand extends FlutterCommand {
   BuildCommand() {
     addSubcommand(new BuildApkCommand());
     addSubcommand(new BuildCleanCommand());
+    addSubcommand(new BuildIOSCommand());
```

Comment:
No, these commands just build the application bundle. Building IPAs is a little bit more involved. When I eventually get around to it though, I can add it as a subcommand to ios.